lib/registry.js

```diff
@@ -16,7 +17,9 @@ export function define(id, spec) {
   if (get(id)) {
     console.warn(`Overwriting spec ${getName(id)}`)
   }
-  spec.name = getName(id)
+  if (!spec.name && typeof spec !== "symbol") {
```

Comment:
`Symbol.for` also creates global symbols. I don't know the implications, but it sounds bad. I'd rather not convert non-symbols to symbols internally, instead enforce using either symbols or strings.